client/cypress/integration/spec.ts

```diff
@@ -0,0 +1,7 @@
+describe('My First Test', () => {
+  it('Visits the initial project page', () => {
+    cy.visit('/')
+    cy.contains('Hello world')
```

Comment:
Could this become a real test (no hello world)?

Reply:
Yes, like described I just created a minimal prototype to check if this feature is welcome. ;)